src/plugins/cloudinary/models/video-source/video-source.js

```diff
@@ -244,10 +244,12 @@ class VideoSource extends BaseSource {
   }
 
   generateRawSource(url, type) {
-    const t = type || url.split('.').pop();
-    const isAdaptive = !!CONTAINER_MIME_TYPES[t];
-    if (isAdaptive) {
-      type = CONTAINER_MIME_TYPES[t][0];
+    type = type || url.split('.').pop();
+
+    const isAdaptive = ['hls', 'dash', 'mpd', 'm3u8'].indexOf(type) !== -1;
```

Comment:
what do you say about moving this list to video source const?

`export const ADAPTIVE_SOURCES = ['hls', 'dash', 'mpd', 'm3u8'];`